src/app/common/components/AppBar/AppBar.tsx

```diff
@@ -73,38 +74,51 @@ const AppBarComponent = () => {
-                              key={index} variant={index === 0 ? 'subtitle1' : 'subtitle1'}
-                              className={classes.button}>
-                              {button.text}
-                            </Typography>
-                          </Tooltip>
-                        );
-                    })}
-            </Grid>
-            <Grid className={classes.secondSection} container spacing={1} justify="flex-end">
-              <Grid item className={classes.walletButton}>
-                <ConnectWalletButton />
+    return (
+      <div className={classes.root}>
+        <AppBar className={classes.bloxColor} position="static">
+          <Toolbar>
+            <Grid container justify={'space-between'} alignItems={'center'}>
+              <Grid item xs={4} md={4}>
+                <Typography onClick={() => { switchPage(config.routes.HOME); }}
+                  variant={'subtitle1'}
+                  className={classes.mainButton}>
+                  ssv.network
+                </Typography>
+              </Grid>
+              <Grid item zeroMinWidth md={3}>
+                <Grid container justify={'space-around'}>
+                  {buttons.map((button: Button, index: number) => {
+                                              return (
+                                                <Grid item key={index}>
+                                                  <Tooltip disableHoverListener={button.active} key={index} disableFocusListener disableTouchListener title="Comming soon...">
+                                                    <Typography onClick={() => { if (button.active) switchPage(button.link, button.newPage); }}
+                                                      variant={'subtitle1'}
+                                                      className={classes.button}>
+                                                      {button.text}
+                                                    </Typography>
+                                                  </Tooltip>
+                                                </Grid>
+                                              );
+                                          })}
+                </Grid>
               </Grid>
-              <Grid item className={classes.menu} onClick={() => { handleClick(!applicationStore.shouldDisplayToolBar); }}>
-                {applicationStore.shouldDisplayToolBar ? <CloseIcon className={classes.menuIcon} /> : <MenuIcon className={classes.menuIcon} />}
+              <Grid item xs={8} md={4}>
+                <Grid container spacing={1} alignItems={'center'} justify={'flex-end'}>
+                  <Grid item>
+                    <ConnectWalletButton />
+                  </Grid>
+                  <Grid item className={classes.menu} onClick={() => { handleClick(!applicationStore.shouldDisplayToolBar); }}>
```

Comment:
Why to not check this condition in `handleClick` itself?

`!applicationStore.shouldDisplayToolBar`

Reply:
because the function not related to view rendering